src/main/java/com/iuc/requests/controller/UserController.java

```diff
@@ -36,7 +46,155 @@ public StaffDto findStaffByMatricule(@RequestParam(value = "userRegistration") S
     }
 
     @PostMapping("/staff")
-    public StaffDto save(@RequestBody StaffDto staffDto) {
-        return userService.save(staffDto);
+    public StaffDto saveStaff(@RequestBody StaffDto staffDto) {
+        return userService.saveStaff(staffDto);
+    }
+
+
+    @DeleteMapping("/staff")
+    public void deleteStaffByMatricule(@RequestParam(value = "userRegistration") String userRegistration){
```

Comment:
Add Regex validator on ``userRegistration``

Reply:
The comment has been taken into consideration